src/OrbitGl/TimeGraph.cpp

```diff
@@ -314,7 +324,7 @@ void TimeGraph::ProcessTimer(const TimerInfo& timer_info, const InstrumentedFunc
   RequestUpdate();
 }
 
-void TimeGraph::ProcessApiEventTimer(const TimerInfo& timer_info) {
+void TimeGraph::ProcessApiEventTimerLegacy(const TimerInfo& timer_info) {
```

Comment:
I'm a little lost, sorry for that. I don't understand why we are changing this name and also why we are having both functions.

Reply:
Thanks for the explanation, I got it now. Feel free to add a bug if we are not planning to tackle this in the following weeks.